app/assets/scripts/components/project-timeline.js

```diff
@@ -17,7 +17,9 @@ var ProjectTimeline = React.createClass({
     const plannedEnd = parseProjectDate(project.planned_end_date);
 
     const actualStart = project.actual_start_date ? parseProjectDate(project.actual_start_date) : null;
-    const actualEnd = project.actual_end_date ? parseProjectDate(project.actual_end_date) : null;
+    const actualEnd = project.actual_end_date
+      ? parseProjectDate(project.actual_end_date)
+      : parseProjectDate(`${new Date().getFullYear()}/${new Date().getMonth() + 1}`) || null;
```

Comment:
@nbumbarger there's no reason to use `parseProjectDate` here and go about re-creating the structure of the date string. You can just use `new Date().getTime()`.